utah-design-system-website/src/react/components/websiteContent/library/components/containers/Accordion/AccordionCssClassesDocumentation.jsx

```diff
@@ -0,0 +1,54 @@
+import {
+  Table, TableBody, TableCell, TableHead, TableHeadCell, TableHeadRow, TableRow, TableWrapper
+} from '@utahdts/utah-design-system';
+
+export function AccordionCssClassesDocumentation() {
+  return (
+    <TableWrapper>
+      <Table className="table--lines-x">
+        <TableHead>
+          <TableHeadRow>
+            <TableHeadCell className="text-left css-classes">CSS Classes</TableHeadCell>
+            <TableHeadCell className="text-left ">Description</TableHeadCell>
+          </TableHeadRow>
+        </TableHead>
+        <TableBody>
+          <TableRow>
+            <TableCell><code>.accordion</code></TableCell>
```

Comment:
in Cards classes the leading dot (.) is removed. should it be removed here?

Reply:
We should make a decision on how to display CSS classes, variables, etc.
Let @jmsharp-utah decide ⚔️

Reply:
Add . to classes